components/rhs_thread/index.ts

```diff
@@ -27,11 +26,10 @@ function makeMapStateToProps() {
         const selected = getSelectedPost(state);
         const socketStatus = getSocketStatus(state);
 
-        let channel = null;
+        const channel = getSelectedChannel(state);
```

Comment:
We can get the selected channel directly from the store instead of having to infer it from the selected post to simplify things slightly